birch

```diff
@@ -213,6 +213,14 @@ cmd() {
             send "TOPIC $chan"
         ;;
 
+        '/away '*)
+            send "AWAY :$a $args"
+        ;;
+
+        '/away'*)
+            send "AWAY"
+        ;;
```

Comment:
Not sure if this could be combined somehow. I tried it, but it prevented the `/away` command without an argument to unset your status being sent properly.